Desktop/exellenteam_python/vingert.py

```diff
@@ -0,0 +1,20 @@
+import random
+import datetime
+
```

Comment:
Please divide your code into functions.
When needed, call them inside a main function.
Remember to document.

Reply:
![Capture](https://user-images.githubusercontent.com/13467749/159838512-4c5218c6-00a5-4778-a520-ba887b39af62.JPG)
This is how it should look like